dockertest/subtestbase.py

```diff
@@ -155,8 +157,15 @@ def failif_not_in(needle, haystack, description=None):
             description = 'string'
         if needle in haystack:
             return
-        raise DockerTestFail("Expected string '%s' not in %s '%s'"
-                             % (needle, description, haystack))
+        expect_s = "Expected string '%s'" % needle
+        if '|' in needle:
+            needles = [n.strip() for n in needle.split('|')]
+            expect_s = "Expected strings %s" % needles
+            for subneedle in needles:
+                if subneedle.strip() in haystack:
+                    return
+        raise DockerTestFail("%s not in %s '%s'" %
+                             (expect_s, description, haystack))
```

Comment:
> ...and having to do that in Every Single Call to self-failif_not_in()

It wouldn't __have__ to be a configuration option, just documented for those cases should be good enough.  Either way, this can easily be added in the future if it's really necessary, so let's just call it "good'nuf" then.